crates/uv/tests/pip_install.rs

```diff
@@ -1302,7 +1302,7 @@ fn install_git_private_https_pat_not_authorized() {
     let context = TestContext::new("3.8");
 
     // A revoked token
-    let token = "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";
+    let token = "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t";
```

Comment:
Why this change? I'll roll back.

Reply:
Sorry, that was a typo

Reply:
Oh, it wasn't actually. It was a weird behaviour of typos pre-commit hook